app/api/user_favourite_events.py

```diff
@@ -79,11 +81,45 @@ class UserFavouriteEventDetail(ResourceDetail):
     """
     User Favourite Events detail by id
     """
+    @jwt_required()
```

Comment:
paranthesis is redundant

Reply:
@iamareebjamal It shows error when not using paranthesis.
![Screenshot from 2019-06-22 14-20-29](https://user-images.githubusercontent.com/31065687/59961715-0c8f5b80-94f9-11e9-9b41-73dd327a9c60.png)

Reply:
OK